README.md

```diff
@@ -11,6 +11,7 @@ Community (college) maintained list of Algorithms and Data Structures implementa
 
 | Algorithm | C | CPP | Java | Python | Golang |
 |:--------------|:----------------:|:----------------:|:----------------:|:-----------------:|:-----------------:|
+| [Bin Sort](www.geeksforgeeks.org/bucket-sort-2)| | |[\[X\]](bin_sort/BinSort.java)| | |
```

Comment:
change the link [www.geeksforgeeks.org/bucket-sort-2](www.geeksforgeeks.org/bucket-sort-2) to [http://www.cdn.geeksforgeeks.org/bucket-sort-2/](http://www.cdn.geeksforgeeks.org/bucket-sort-2/)
as the previous one is not working